Client/src/Components/NavCss.css

```diff
@@ -11,6 +11,7 @@
     justify-content: space-between;
     align-items: center;
     padding: 20px 20px 25px; 
+    border-bottom: 0.5px solid rgb(165, 165, 165) ;
```

Comment:
To maintain a clean and maintainable codebase, consider moving inline styles like 'border-bottom' to the CSS class definitions. [medium]